frodo.js

```diff
@@ -1,130 +1,130 @@
 /**
```

Comment:
SCRIPT-The CSS rules here seem to be redundant. Can you please review and remove unnecessary ones?